.github/workflows/build_wheels.yml

```
@@ -206,6 +208,9 @@ jobs:
             python3 \
             python3-devel \
             python3-debug
+          dnf debuginfo-install -y \
+            python3-libs \
+            python3
```

Looking at the log apparently listing both is redundant:

```suggestion
            python3
```